spec/index.html

```diff
@@ -563,109 +906,165 @@ <h3>Algorithm</h3>
                   <a>identifier issuer</a> initialized with the prefix
                   <code>_:b</code>.</li>
-                <li id="ca-6-2-3">Use the
+                <li id="ca-5-2-3">Use the
                   <a href="#issue-identifier">Issue Identifier algorithm</a>,
                   passing <var>temporary issuer</var> and <var>n</var>, to
                   issue a new temporary <a>blank node identifier</a> <var>b<sub>n</sub></var>
                   to <var>n</var>.</li>
-                <li id="ca-6-2-4">Run the
+                <li id="ca-5-2-4">Run the
                   <a href="#hash-nd-quads">Hash N-Degree Quads algorithm</a>,
-                  passing <var>temporary issuer</var>, and append the
+                  passing the <a>canonicalization state</a>,
+                  <var>n</var> for <var>identifier</var>, and
+                  <var>temporary issuer</var>,
+                  appending the
                   result to the <var>hash path list</var>.</li>
               </ol>
             </li>
-            <li id="ca-6-3">For each <var>result</var> in the <var>hash path list</var>,
+            <li id="ca-5-3">For each <var>result</var> in the <var>hash path list</var>,
               <a>code point ordered</a> by the <var>hash</var> in <var>result</var>:
+              <details>
+                <summary>Explanation</summary>
+                <p>The previous step created temporary identifiers for the
+                  blank nodes sharing a common first degree hash,
+                  which is now used to generate their canonical identifiers.</p>
+              </details>
               <ol>
-                <li id="ca-6-3-1">For each <a>blank node identifier</a>,
+                <li id="ca-5-3-1">For each <a>blank node identifier</a>,
                   <var>existing identifier</var>, that was issued a temporary
                   identifier by <var>identifier issuer</var> in <var>result</var>, issue
                   a canonical identifier, in the same order, using the
                   <a href="#issue-identifier">Issue Identifier algorithm</a>,
                   passing <a>canonical issuer</a> and
                   <var>existing identifier</var>.
+                  <p class="ednote">
+                    An additional explanation
+                    should be added, because an unusual thing is happening here.
+                    The explanation in <a href="#ca-5-3">5.3</a>)
+                    says "temporary identifiers for the blank nodes sharing a common first degree hash".
+                    The reader would expect then issuing a canonical id for that blank node in some manner,
+                    and that is it.
+                    However, <a href="#ca-5-3-1">5.3.1</a>) actually assigns several canonical ids:
+                    the text says that one has to go through each identifier that is in the issuer.
+                    That situation should get focus and explanation on this level.
```

Comment:
I had trouble parsing this note -- but I think what it's saying is that more than one blank node is assigned a canonical label (because each of the blank nodes that appear in the temporary issuer is given a label in the same order that they were visited). Regardless, I agree we could add some text that explains this further.

Reply:
That was just a copy of @iherman's comment as a note to come back and improve this as a `detail`.

Reply:
@dlongley look at latest update (in 5f915f1e1b4e40136b1c98d5a7759f09a28e2969) for the added details. If you're happy with this, I'll go ahead and merge this PR.

Reply:
@gkellogg LGTM.